package.json

```diff
@@ -176,7 +178,13 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "npm run lint -- --errors"
+      "pre-commit": "lint-staged"
     }
+  },
+  "lint-staged": {
+    "*.js": [
+      "vjsstandard --fix --errors",
```

Comment:
looks like `--errors` can't be combined with `--fix`. Removing it makes it fix things.